ui/narrative/methods/import_fastq_sra_as_reads_from_staging/spec.json

```diff
@@ -113,6 +113,7 @@
       "field_type" : "text",
       "text_options" : 
       {
+        "valid_ws_types": [ KBaseFile.SingleEndLibrary", "KBaseFile.PairedEndLibrary"]
```

Comment:
spec file is invalid. I think it's due to missing comma between valid_ws_types and is_output_name.